Service/Endpoint/EndpointFactory.php

```diff
@@ -25,10 +25,12 @@ public function createEndpoint(
         string $requestClass,
         string $responseFormat,
         $responseClass = null,
-        $dateTimeFormat = null
+        $dateTimeFormat = null,
+        string $errorClass = null
```

Comment:
please, make the constructor a bit more consistent, not just append in the end like it was forgotten 😅  : 
request Format/Class 
response Format/Class
**error Format/Class**
dateTime